pincer/objects/events/voice.py

```diff
@@ -24,6 +25,122 @@ class VoiceServerUpdateEvent(APIObject):
     :param endpoint:
         the voice server host
     """
+
     token: str
     guild_id: Snowflake
     endpoint: Optional[str] = None
+
+
+@dataclass
+class VoiceChannelSelectEvent(APIObject):
+    """
+    Sent when the client joins a voice channel
+
+    :param channel_id:
+        id of channel
+
+    :param guild_id:
+        id of guild
+    """
+
+    channel_id: Optional[Snowflake] = None
```

Comment:
Change all to `varname: APINullable[type] = MISSING`

Reply:
The channel_id arg will always be passed, so it wont be APINullable, but the channel_id could be None, that's why I added that